src/utils/validate-object-id-hook.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const errors = require('feathers-errors');
+const mongoose = require('mongoose');
+
+var validateObjectId = function() {
+  return function(hook) {
+    const id = hook.id;
+    if (id && !mongoose.Types.ObjectId.isValid(id)) {
```

Comment:
I wonder. Should the ObjectId type we defined in model already handle it?
Also, the error message said 'no record found' but we actually just validate the objectId.